Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/events/DefaultTraceLocation.kt

```diff
@@ -14,7 +18,7 @@ data class DefaultTraceLocation(
     override val startDate: Instant?,
     override val endDate: Instant?,
     override val defaultCheckInLengthInMinutes: Int?,
-    override val signature: String,
+    override val signature: ByteString,
```

Comment:
`CheckIn` also contains the signature, so we should also change it to `ByteString` there. However, these refactoring are getting out of scope for this PR, so I can do this in my "Store binary representation of SignedTraceLocation in database" Task if you like?

Reply:
> CheckIn also contains the signature, so we should also change it to ByteString there. 

Yes

> However, these refactoring are getting out of scope for this PR, so I can do this in my "Store binary representation of SignedTraceLocation in database" Task if you like?

Yes please.